WickedEngine/shaders/shadowHF.hlsli

```diff
@@ -107,4 +107,29 @@ inline float shadow_2D_volumetricclouds(float3 P)
 	return 1.0;
 }
 
+// Sample light and furthest cascade for large mediums (volumetrics)
+// Used with SkyAtmosphere and Volumetric Clouds
+inline void furthest_cascade_volumetrics(inout ShaderEntity light, inout uint furthestCascade)
```

Comment:
Can this entity index be computed on CPU instead? Seems like a waste to run this loop in a shader. Or did you consider using the `GetWeather().most_important_light_index` ? The most important light index is the first directional light in the light array.

Reply:
Oh yes that's way better, have no idea why I didn't notice. I pushed a fix. I should even have made it run on the CPU instead regardless, my bad

Reply:
Yes, it is offset into the light array, so it needs light offset into entity array. Just like how you did it.